user-service/src/main/java/com/movie/recommender/user/controller/UserController.java

```diff
@@ -33,5 +33,30 @@ public ResponseEntity<String> login(@Valid @RequestBody UserLoginDTO userLoginDT
         String token = userService.login(userLoginDTO.getUsername(), userLoginDTO.getPassword());
         return ResponseEntity.ok(token);
     }
+
+    @GetMapping("/{userId}/favorites")
```

Comment:
add the request path `/api/v1/favorites/` at the class level in `FavoritesController` to simplify method mappings